web_frontend/cloudscheduler/glintwebui/glint_views.py

```diff
@@ -969,54 +1014,53 @@ def upload(request, group_name=None):
                     "target_cloud_name": cloud,
-                    "image_name":        image_file.name,
-                    "image_id":          image.id,
-                    "checksum":          image.checksum,
-                    "requester":         active_user.username,
+                    "image_name": image_file.name,
+                    "image_id": image.id,
+                    "checksum": image.checksum,
+                    "requester": active_user.username,
                 }
                 config.db_merge(IMAGE_TX, tx_req)
                 config.db_commit()
                 logger.info("Transfer id:%s queued" % tx_id)
                 tx_request.apply_async((tx_id,), queue='tx_requests')
 
-        #return to project details page with message
-        msg="Upload successfully queued, returning to images..."
+        # return to project details page with message
+        msg = "Upload successfully queued, returning to images..."
         where_clause = "group_name='%s' and cloud_type='%s'" % (group_name, "openstack")
         rc, qmsg, cloud_list = config.db_query(CLOUDS, where=where_clause)
         context = {
-                'group_name': group_name,
-                'cloud_list': cloud_list,
-                'max_repos': len(cloud_list),
-                'redirect': "true",
+            'group_name': group_name,
+            'cloud_list': cloud_list,
+            'max_repos': len(cloud_list),
+            'redirect': "true",
 
-                #view agnostic data
-                'active_user': active_user.username,
-                'active_group': active_user.active_group,
-                'user_groups': active_user.user_groups,
-                'response_code': rc,
-                'message': msg,
-                'is_superuser': active_user.is_superuser,
-                'version': config.get_version()
+            # view agnostic data
+            'active_user': active_user.username,
+            'active_group': active_user.active_group,
+            'user_groups': active_user.user_groups,
+            'response_code': rc,
+            'message': msg,
+            'is_superuser': active_user.is_superuser,
+            'version': config.get_version()
         }
         config.db_close()
         return render(request, 'glintwebui/upload_image.html', context)
 
     elif request.method == 'POST' and request.POST.get('myfileurl'):
         if group_name is None:
-
             logger.error("No group name, using user's default")
             group_name = active_user.active_group
 
-
-        #download the image
+        # download the image
         img_url = request.POST.get('myfileurl')
         image_name = img_url.rsplit("/", 1)[-1]
         logger.info("File to upload: %s" % image_name)
-        file_path = config.categories["glintPoller.py"]["image_cache_dir"] + image_name # This file will have to be renamed with the checksum after uploading to a cloud
+        file_path = config.categories["glintPoller.py"][
```

Comment:
no need to break up this line, instead the comment should be moved to a line of it's own.